app/components/ToggleOption/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+*
+* ToggleOption
+*
+*/
+
+import React from 'react';
+import { FormattedMessage } from 'react-intl';  // eslint-disable-line import/no-unresolved
+
+function ToggleOption(props) {
+  return (
+    <option value={props.value}>
+      <FormattedMessage {...props.message} />
```

Comment:
of course! normally i would have opened PRs for these but i was beyond exhausted last night, just wanted to review everything as promised :) will try to find some time later today to address these